### ISSUE TITLE
Hacktoberfest - move from --name parameter to positional arg

### ISSUE THREAD

**Issue by alexellis** [openfaas/faas-cli#133]
## Expected Behaviour

```
faas-cli new mychatbot --lang python
```

## Current Behaviour

```
$ faas-cli new --name mychatbot --lang python
```

## Possible Solution

Match the `remove` verb

## Context

Helping to move the developer-experience forward.

**Comment by gardlt** [comment]
Is this issue, still open, i would like to give it a go?

**Comment by alexellis** [comment]
I've just merged this one @gardlt but I have another one almost exactly the same 👍 

Feel free to start on: #134 and ping alex@openfaas.com to join the Slack community

**Comment by alexellis** [comment]
Fixed in #132
